### ISSUE TITLE
[Feature request] m.youtube.com support

### ISSUE THREAD

**Issue by KaKi87** [zpix1/yt-anti-translate#16]
Thanks

**Comment by zpix1** [comment]
Do you use this on mobile? Which browser?

**Comment by KaKi87** [comment]
@zpix1 :pray:

**Comment by NeonMan** [comment]
I make my desktop browser pretend it is a mobile browser so Youtube serves me the more lightweight m.youtube instead, and I am having the same issue where video titles are not being untranslated.

**Comment by Cherub5021** [comment]
> Do you use this on mobile? 
Currently, it is not possible to switch audio tracks between the original and dubbed versions in mobile browsers.

> Which browser?
Here are the ones that support extensions on mobile devices:  
- Firefox (you can enable desktop site mode to install unavailable extensions)  
- Kiwi browser  
- Edge  
- [Ultimatum](https://github.com/gonzazoid/Ultimatum/releases)  
- Yandex browser

**Comment by RayRJJackson** [comment]
It would be nifty for the extension to be available on mobile browsers like Firefox.

**Comment by RayRJJackson** [comment]
>     * Firefox (you can enable desktop site mode to install unavailable extensions)

Tried it. It's installed but it doesn't work. So yeah, it still needs m.youtube support.

**Comment by zpix1** [comment]
Currently firefox extension version 1.19 is not published, while 1.19 should work on mobile.

**Comment by zpix1** [comment]
> Just tried the new version of the extension on Firefox mobile.
> It doesn't work properly. The audio is still AI-dubbed.
> The titles and the description are untranslated like expected, but the chapters and the audio are still auto-translated.
> The options in the extension are all checked, so it should untranslate, but it's not working.

Yes, audio features are not working on mobile for now. I will check what we can do.
